relay-server/src/utils/multipart.rs

```diff
@@ -177,7 +215,10 @@ where
             let content_type = field.content_type().cloned();
             let field = LimitedField::new(field, config.max_attachment_size());
             match field.bytes().await {
-                Err(multer::Error::FieldSizeExceeded { .. }) if ignore_large_fields => continue,
+                Err(multer::Error::FieldSizeExceeded { limit, .. }) if ignore_large_fields => {
```

Comment:
Cheating here a bit by putting the consumed size into the error rather than the limit that is breached such that we can use that value to generate the outcome.